websites/T/TMBW/metadata.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.13",
+  "apiVersion": 1,
+  "author": {
+    "name": "Anant",
+    "id": "6629851641579044874"
+  },
+  "service": "TMBW",
+  "description": {
+    "en": "Displays what article you're reading on the They Might Be Giants fan-maintained wiki, TMBW.net."
+  },
+  "url": [
+    "tmbw.net",
+    "www.tmbw.net"
+  ],
+  "version": "1.0.0",
+  "logo": "https://i.ibb.co/Y7s4FFHp/TMBW-512x512.png",
```

Comment:
This logo is pretty low-resolution. Find a higher-resolution version or alternative logo (for example, https://tmbw.net/share.png)